gnucash/gnome/gnc-plugin-page-register.c

```diff
@@ -302,10 +309,9 @@ static GncInvoice* invoice_from_split (Split* split);
 #define PASTE_TRANSACTION_TIP            N_("Paste the transaction from the clipboard")
 #define DUPLICATE_TRANSACTION_TIP        N_("Make a copy of the current transaction")
 #define DELETE_TRANSACTION_TIP           N_("Delete the current transaction")
-#define ASSOCIATE_TRANSACTION_TIP        N_("Update Association for the current transaction")
-#define ASSOCIATE_TRANSACTION_OPEN_TIP   N_("Open Association for the current transaction")
-#define ASSOCIATE_TRANSACTION_REMOVE_TIP N_("Remove the association from the current transaction")
-#define JUMP_ASSOCIATED_INVOICE_TIP      N_("Open the associated invoice")
+#define ASSOCIATE_TRANSACTION_TIP        N_("Manage the document associated with the current transaction")
+#define ASSOCIATE_TRANSACTION_OPEN_TIP   N_("Open the associated document for the current transaction")
+#define JUMP_ASSOCIATED_INVOICE_TIP      N_("Jump to the associated invoice")
 #define CUT_SPLIT_TIP                    N_("Cut the selected split into clipboard")
 #define COPY_SPLIT_TIP                   N_("Copy the selected split into clipboard")
 #define PASTE_SPLIT_TIP                  N_("Paste the split from the clipboard")
```

Comment:
Tips should be full sentences with full stop.

Reply:
OK, but few if any are, so they should all be done in a single commit and that should be done at the same time as the ellipsis change.

Reply:
For this section yes. But my fear is, there could be other sections containing e.g. only an infinitive instead of a full sentence. Then it is not trivial.

Reply:
Well, you have ~2 years to find them. ;-)